commands.go

```diff
@@ -24,27 +24,28 @@ func Commands(metaPtr *command.Meta) map[string]cli.CommandFactory {
 	}
 
 	return map[string]cli.CommandFactory{
-		"setup-omm": func() (cli.Command, error) {
-			return &command.InstallMayaCommand{
-				M: meta,
-			}, nil
-		},
-		"setup-osh": func() (cli.Command, error) {
-			return &command.InstallOpenEBSCommand{
-				M: meta,
-			}, nil
-		},
-		"omm-status": func() (cli.Command, error) {
-			return &command.ServerMembersCommand{
-				Meta: meta,
-			}, nil
-		},
-		"osh-status": func() (cli.Command, error) {
-			return &command.NodeStatusCommand{
-				Meta: meta,
-			}, nil
-		},
-
+		/*
```

Comment:
Include a comment referring to the reason for commenting this code.

Reply:
Remove (blank) extra space also.